src/System.IO.Pipes.AccessControl/tests/System.IO.Pipes.AccessControl.Tests.csproj

```diff
@@ -2,6 +2,7 @@
   <PropertyGroup>
     <ProjectGuid>{A0356E61-19E1-4722-A53D-5D2616E16312}</ProjectGuid>
     <Configurations>netcoreapp-Windows_NT-Debug;netcoreapp-Windows_NT-Release;uap-Windows_NT-Debug;uap-Windows_NT-Release</Configurations>
+    <AssemblyBeingTestedName Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'netstandard'">System.IO.Pipes</AssemblyBeingTestedName>
```

Comment:
@pjanotti sorry Paulo to bother you. If you have time it would be great if you could answer a few minor questions:

1. Is the `TargetGroup == netstandard` condition necessary here? Why not for uap?
2. Should we still keep the TestRuntime property for faster code coverage runs with coverlet or doesn't it really matter with it anymore?

And just to let you know, I'm migrating the CodeCoverage targets right now to arcade and with that get rid of OpenCover and make Coverlet the default runner. My plan is to register both Coverlet and ReportGenerator as global tools.

Reply:
And I guess this test assembly tests against System.IO.Pipes and not System.IO.Pipes.AccessControl, therefore we need this property?

Reply:
Hi @ViktorHofer  as you guessed I've been busy with other stuff but let me see if I can help...

> Is the TargetGroup == netstandard condition necessary here?

Yes, due to the reason that you described: the S.IO.P.AccessControl doesn't have the implementation anymore (it migrated to S.IO.P) but it is part of the standard, so we need S.IO.P on netstandard.

> Should we still keep the TestRuntime property for faster code coverage runs with coverlet or doesn't it really matter with it anymore?

Yes, Coverlet does IL rewrite so to avoid leaving the slower instrumented CoreLib being used in case of any error. Also, I don't remember exactly but I __think__ that Coverlet wasn't able to instrument the native CoreLib which required us to get the IL version anyway (I am almost sure of it).

Reply:
Ok great thanks a lot Paulo. One remaining question, what is the reason that we need the coverage dedicated runtime directory with Coverlet? I understand that assemblies are instrumented and therefore its content is changed and we don't want that but Coverlet backups such assemblies before the instrumentation and restores it after the coverage measurement.

Reply:
The Private.CoreLib that comes on the default folder is crossgen, IIRC Coverlet cannot handle crossgen files and as such we need the IL version, Coverlet then restores the IL version not the original crossgen version. So the risk that people wanted to avoid was someone ending up running the IL version of CoreLib when they intended to run the crossgen (the IL one is a bit slower). I think there is a bit of this history on the issues related to coverage on buildtools and corefx.

Reply:
Ok thanks again Paulo. I think that's all I need to know. I'll appreciate your help.